subsys/bluetooth/host/hci_core.c

```diff
@@ -228,18 +228,59 @@ static void handle_vs_event(uint8_t event, struct net_buf *buf,
 	/* Other possible errors are handled by handle_event_common function */
 }
 
-#if defined(CONFIG_BT_HCI_ACL_FLOW_CONTROL)
-void bt_hci_host_num_completed_packets(struct net_buf *buf)
+enum acl_data_flags *bt_get_acl_buf_flags(struct net_buf *acl_packet)
 {
+	return &(acl(acl_packet)->flags);
+}
 
+void bt_send_one_host_num_completed_packets(uint16_t handle)
+{
+#if !defined(CONFIG_BT_HCI_ACL_FLOW_CONTROL)
+	ARG_UNUSED(handle);
+	return;
+#else
```

Comment:
```suggestion
	if (!IS_ENABLED(CONFIG_BT_HCI_ACL_FLOW_CONTROL)) {
		return;
	}
```

Reply:
`*cracks knuckles*` come at me clang linker
(I'll try)

Reply:
Oh, are some of the other parts in this function guarded? 

Might not be possible then :)